src/sections/collection/datasets-list/file-card/FileCard.module.scss

```diff
@@ -0,0 +1,75 @@
+@use 'sass:color';
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module";
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/typography.module";
+
+.container {
+  margin: 6px 0;
+  padding: 4px 10px;
+  border: 1px solid $dv-file-border-color;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.title {
+  display: flex;gap: 8px;
+}
+
+.icon {
+  margin-top: 2px;
+  font-size: 1.3em;
+  line-height: 1.1;
+
+  > div >span {
+    margin-right: 0;
+  }
+}
+
+.thumbnail {
+  width: 48px;
+  margin: 8px 12px 6px 0;
+  font-size: 2.8em;
+
+  img {
+      vertical-align: top;
+  }
+}
+
+.info {
+  display: flex;
+  color: $dv-subtext-color;
+}
+
+.description {
+  display: flex;
+  font-size: $dv-font-size-sm;
+}
+
+.descriptionText {
+  display: -webkit-box; -webkit-line-clamp: 3;  line-clamp: 3; -webkit-box-orient: vertical;
+  flex-direction: column;
```

Comment:
we don't need `flex-direction: column `here, I think it was meant to be used only for the main container, for the one I suggest in previous comment to rename as `.card-info-container`